cape/cape_result.py

```diff
@@ -2243,7 +2287,30 @@ def process_buffers(
                     ):
                         buffer_body.append(table_row)
                         count_per_source_per_process += 1
-
+                        network_buffers.append((process, api_call, buffer))
+
+    network_buffers_PE = []
+    for process, api, buffer in network_buffers:
+        if any(PE_indicator.decode('ascii') in buffer for PE_indicator in PE_INDICATORS):
+            hash = sha256(buffer.encode()).hexdigest()
+            network_buffers_PE.append((f"{str(process)}-{api}-{hash}", buffer))
+    buffers_PE = []
+    for process, api, buffer in buffers:
+        if any(PE_indicator.decode('ascii') in buffer for PE_indicator in PE_INDICATORS):
+            hash = sha256(buffer.encode()).hexdigest()
+            buffers_PE.append((f"{str(process)}-{api}-{hash}", buffer))
+
+    if not os.path.exists(BUFFER_PATH):
+        os.mkdir(BUFFER_PATH)
+
+    for filename, buffer in network_buffers_PE:
+        with open(f"{BUFFER_PATH}/{filename}.txt", "w") as f:
```

Comment:
The files should not have a .txt extension since they are PEs. No extension is preferred since we don't know what the file type is other than ~ PE ~
```suggestion
        with open(f"{BUFFER_PATH}/{filename}", "w") as f:
```